### PR TITLE
feat: pagination for media in search

### PR DESCRIPTION
**Pull request by MarLaslo**
Implementation of bagination. 
Also I cleaned a code a little bit. 

I have removed the filter part from the search page as it was completely useless.

### CODE REVIEW

**Comment by sofushn** [comment]
@IdaHay do you know if this has been implemented in one of your prs?

**Comment by idax6797** [comment]
> @IdaHay do you know if this has been implemented in one of your prs?

No, I have not created paging on the search yet. So if this is set up then it is nice.

File: src/frontend/src/services/mediaService.js [hunk 1]
```diff
@@ -96,29 +96,24 @@ export const fetchReleases = async (id) => {
 };
 
 export const fetchMedia = async ({
-  page = 1,
-  count = 18,
+  page,
+  pageCount,
```
Comment (idax6797) [inline src/frontend/src/services/mediaService.js:100]: I think we should follow the endpoints naming, so count and not pageCount

File: src/frontend/src/pages/SearchPage.jsx [hunk 2]
```diff
@@ -1,78 +1,58 @@
...
+
+    const getMedia = async () => {
       try {
-        const response = await fetchMedia({ query: searchQuery, queryType });
-        const resultList = response.items;
-        setResults(resultList);
-        applyFilters(resultList, filterCriteria);
+        const resultList = await fetchMedia({
+          page: currentMediaPage,
+          pageCount: mediaItemsPerPage,
```
Comment (idax6797) [inline src/frontend/src/pages/SearchPage.jsx:40]: use count, as the endpoint naming

File: src/database
Comment (idax6797) [inline src/database:1]: Should this be committed?

File: src/frontend/src/pages/SearchPage.jsx [hunk 3]
```diff
@@ -1,78 +1,58 @@
...
+          query: searchQuery,
+          queryType,
+        });
+        setMediaResults(resultList.items);
+        setTotalItems(resultList.numberOfItems);
       } catch (error) {
         console.error('Error searching:', error);
```
Comment (idax6797) [inline src/frontend/src/pages/SearchPage.jsx:47]: Could a toast also explain that this went wrong? for better UX

**Review by idax6797 — DISMISSED** [review]

File: src/database
Comment (MarLaslo) [reply src/database:1]: I have no idea what is that docker thing...

Comment (sofushn) [reply src/database:1]: This is just the reference to which version of database repo is used, if its pointing at the latest version of main its fine

**Review by sofushn — APPROVED** [review]

**Pull request merged**